packages/tradershub/src/flows/FirstSignup/FirstSignupWrapper/FirstSignupWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import { Form, Formik } from 'formik';
+import ReactModal from 'react-modal';
+import { CUSTOM_STYLES } from '@/helpers';
+import { SignupScreens } from '../SignupScreens';
+
+export type TFirstSignupFormValues = {
+    citizenship: string;
+    country: string;
+    password: string;
+};
+
+const FirstSignupWrapper = () => {
+    const [isOpen] = useState(false);
```

Comment:
there's no setter for this state to handle the opening & closing of the ReactModal
How are we handling the visibility here?